README.md

```diff
@@ -33,6 +33,15 @@ You can configure the behaviour and output of the `tldr` client by setting envir
     * If set to `0`, the client will fetch from the internet, and fall back to the cache if the page cannot be fetched from the internet.
 * `TLDR_CACHE_MAX_AGE` (default is `24`): maximum age of the cache in hours to be considered as valid when `TLDR_CACHE_ENABLED` is set to `1`.
 
+#### Cache location in order of precedence
+
+* `$XDG_CACHE_HOME/tldr`
+* `$HOME/.cache/tldr`
+* `~/.cache/tldr`
+* Previously, the cache resided in `$HOME/.tldr_cache`
+
+If you are experiencing any issues with *tldr*, deleting the cache files might resolve them.
```

Comment:
I think "any issues" may be a bit too broad, no? I mean, not all problems will be related to the cache folder...

Reply:
Well, it says "any issue might be solved by", i.e. if something breaks, try deleting the cache, because it's an easy fix. Only if that doesn't work, go looking for other fixes.

Also, I didn't know how I would phrase the range of issues that might be caused by the cache.

Reply:
Maybe something less final, then? E.g. "If you are experiencing issues with *tldr*, consider deleting the cache files." The distinction is subtle, but I think it sounds less definitive / assuring.